src/utils/match-collection.ts

```diff
@@ -3,15 +3,15 @@ import { isMatch } from 'lodash';
 // NOTE: this method will duplicate the matching items
 // currently cookies-tests use incorrect behavior of this method
 // the method and cookies-tests should be rewritten
-export default function matchCollection (items: object[], filters: object[]): object[] {
+export default function matchCollection (items: object[], filters: object[], customFilter?: Function): object[] {
```

Comment:
Can we rewrite this function as follows:
```js
export default function matchCollection (items: object[], filters: object[], matchFn = isMatch): object[] {
...
if (matchFn(item, filter))
    result.push(item);
...
```

Reply:
Sure